api/alias_api/__init__.py

```diff
@@ -196,6 +196,7 @@ def import_alias_api():
 
     try:
         alias_api = importlib.util.module_from_spec(spec)
+        sys.modules["alias_api"] = alias_api
```

Comment:
Just curious why this line needed to get moved up here?

Reply:
I think I changed this when I did my tests following the official guide: https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly
but not sure if it changes something to the behavior

Reply:
Interestingly, the [examples ](https://docs.python.org/3.7/library/importlib.html#examples)for 3.7 say that adding the module to `sys.modules` is optional. So I looked into `module_from_spec` more, and it already adds the module to `sys.modules`. Either way though, doesn't hurt to leave it in to make sure it is set in modules.